src/medVtkInria/vtkDataManagement/vtkMetaDataSetSequence.cxx

```diff
@@ -1027,6 +1027,24 @@ double* vtkMetaDataSetSequence::GetCurrentScalarRange()
   return val;
 }
 
+double* vtkMetaDataSetSequence::GetCurrentScalarRange(const QString & attributeName)
```

Comment:
This function and the one above differ by only one minor element. This duplication should be removed. I see two options:
1. Merge both functions into one with the signature ```GetScalarRange(QString attributeName = QString())``` and if the string is empty then you take the current scalar array.
2. Keep separate functions (but naming the second one ```GetScalarRange``` for the reasons I mentioned in my comment on ```vtkMetaDataSet```) but in the source file both these functions call a private one that is similar to the one I describe in option 1.